src/js/experimental/media-chrome-listitem.js

```diff
@@ -11,6 +11,7 @@ template.innerHTML = `
     padding: 0.5em;
     margin: 0em;
     cursor: pointer;
+    white-space: nowrap;
```

Comment:
I was looking at this the other way. I'm thinking that we may want to provide a css variable for this because otherwise, if someone sets a max width, it'll scroll and maybe not everyone wants that.